src/site/content/en/angular/faster-angular-change-detection/index.md

```diff
@@ -126,14 +126,7 @@ With the two instances of `EmployeeListComponent`, the app forms the following c
 
 When the user begins entering the name of a new employee in the input box in an`EmployeeListComponent`, Angular triggers change detection for the entire component tree starting from `AppComponent`. This means that while the user is typing in the text input, Angular is repeatedly recalculating the numeric values associated with each employee to verify that they haven't changed since the last check.
 
-To see how slow this can be, try entering the name of a new employee in the app below:
-
-<div class="glitch-embed-wrap" style="height: 480px; width: 100%;">
-  <iframe
-    src="https://stackblitz.com/github/mgechev/change-detection-web-dev?embed=1&file=src/app/app.component.ts&view=preview"
-    style="height: 100%; width: 100%; border: 0;">
-  </iframe>
-</div>
+To see how slow this can be, open the non-optimized version of the project on [StackBlitz](https://stackblitz.com/github/mgechev/change-detection-web-dev) and try entering an employee name.
```

Comment:
```suggestion
To see how slow this can be, open the [non-optimized version of the project on StackBlitz](https://stackblitz.com/github/mgechev/change-detection-web-dev) and try entering an employee name.
```

Reply:
Moved the square bracket to link more of the text so it is clear we're not just linking to stackblitz.com.